ctrd/image.go

```diff
@@ -38,16 +38,17 @@ func (c *Client) ListImages(ctx context.Context, filter ...string) ([]types.Imag
 	}
 
 	images := make([]types.Image, 0, 32)
-	digestPrefix := "sha256:"
 	for _, image := range imageList {
 		descriptor := image.Target()
-		digest := []byte(descriptor.Digest)
+		dig := descriptor.Digest
```

Comment:
the name "digest" seems to be better than "dig" ???

Reply:
ok! wil do

Reply:
Have solved. Please take a look! thanks